.github/workflows/release.yml

```
@@ -20,7 +20,7 @@ jobs:
 
     - name: Release
       id: release
-      uses: cycjimmy/semantic-release-action@v4
+      uses: cycjimmy/semantic-release-action@9a24fc62fd6dbf0c523b81421b634ee3c2c2ad55
```

Might be a good idea to add the version at the end of the hash, like the other actions, to make it easier to know which version we are using.
```suggestion
      uses: cycjimmy/semantic-release-action@9a24fc62fd6dbf0c523b81421b634ee3c2c2ad55 # v4
```